### DISCUSSION TITLE
Integration with path guiding libraries

### DISCUSSION THREAD

**Discussion by saeedhd96** [mitsuba-renderer/mitsuba3#907] [Q&A]
Hello,

I was wondering if there is any plan to integrate Mitsuba3 with path guiding libraries such as [Intel Open PGL](https://www.intel.com/content/www/us/en/developer/articles/technical/open-path-guided-rendering-made-easier.html) ([github](https://github.com/OpenPathGuidingLibrary/openpgl)).

Thanks,
-Saeed

**Comment by wjakob** [comment]
We don't currently have a plan to pursue this. It might be quite hard to do so since Mitsuba is based on its own custom Jit-compiler.
